openweathermap-ios/openweathermap-ios/BookmarkLocation/Model/BookmarkLocation.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BookmarkLocation.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import Foundation
+
+private enum BookmarkLocationKeys: String {
+    case title, latitude, longitude
+}
+
+final class BookmarkLocation: NSObject, NSCoding {
+    let title: String?
+    let latitude: String?
+    let longitude: String?
```

Comment:
Why are all optional??? IMHO, you should save and restore only healthy data.

Reply:
Fixed in b6bb083